### ISSUE TITLE
It's a me, the original guy

### ISSUE THREAD

**Issue by Tiamarth** [drakkar1969/Arc-for-Vivaldi#1]
I hope you don't mind me creating this issue here but I was unsure how else to contact you. I'm back and was planning on picking this project back up but I see you've got a fairly complete fork here and my code seems pretty heavily broken on newer Vivaldi versions. Basically I just want to know how much your fork has changed, whether or not it's worth my time to pick my original project back up. I do see that you've dropped support for Arc Grey, so that is one thing I could continue.

**Comment by drakkar1969** [comment]
Sorry for the delayed response ...
First off, thanks so much for the great work on the original Vivaldi Arc theme, great theme - and without it I would never have had the courage to start from scratch.
I would estimate I have changed 60 to 70% of the css code, so really up to you how you want to take this forward. I mean I would be happy for you to merge everything back into your original project and I can contribute there - assuming you're ok with what I've done that is ...
What's missing/dropped in my fork at this point is the Arc Grey theme, Opera panel and support for the Arc-Darker theme. Plus the window buttons are a little bit "hacky".

**Comment by Tiamarth** [comment]
Aw, thanks for the kind words.

tl;dr: Your css is prettier than mine, and I prefer it based on that alone. I'm open to the idea of merging the repositories but would have to test your css more to make sure I like all of the changes you made.

You took long enough to respond that I wasn't sure if you were going to and went ahead and updated my original skin. It's working now, and I took a page from your book and added color variables. I would potentially be interested in merging the repositories, but I think I'd first have to test yours more thoroughly to make sure I prefer it. Some things I can say without having tested it much are:

Your css looks way cleaner than mine and the comments are, I think, much clearer. If I were someone looking at our two versions of this skin and was too lazy to read the css, I'd install yours over mine just based on its rough appearance lol.

Your install instructions only include Linux, which is fair, I mean Arc *is* a GTK theme. But I've had people install it on other platforms and I want to support them too.

Arc Darker is my personal favourite variation of Arc, so I'll definitely be maintaining support for it. When I originally wrote the skin I wrote it for Arc Darker and then only added support for the other two when I was about ready to share it with people.

I also have friends who prefer black and grey themes over dark blue ones, so I'll be maintaining support for Arc Grey as well even though it looks like the actual gtk theme has been discontinued.

No worries about the opera panel css, it never should have been in the same repository as afv but giving its own repository seemed like overkill for, like, 30 lines of css. I plan on removing the features from afv that do anything other than make Vivaldi use Arc's colors and moving them to their own project. (Including opera panel, the 1px window border, white space reduction, merging the address bar and title bar when tabs aren't at the top, etc.) Although I will, of course, make sure it is compatible with afv.

And the window buttons have always been hacky. I can do them with javascript instead, much cleaner, but I prefer to keep everything in one css file unless absolutely necessary. It's easier for people to install that way.

**Comment by Tiamarth** [comment]
I've been using yours now for a few hours and, other than the obviously missing Arc Darker and Arc Grey variations, I'm not noticing any real differences in appearance or function. You don't seem to have hover transitions for buttons in the ui but that's not really an issue. When I updated mine the other day I made it so the hover transitions only work if the user has animations enabled in their Vivaldi settings, since there's a class now that makes it possible.

I don't plan on removing the animations, though, as I think they're a nice bit of eye candy. So I'm still open to the idea of merging the repositories but I think there's enough taste differences to justify keeping the projects separate to provide people with alternative options. Let me know what you think.

I do still like your css structure and style more than mine, though.

**Comment by Tiamarth** [comment]
I don't mean to pressure you to reply or annoy you but it's been a few more days and I see you've re-added support for Arc Darker, and after reviewing some of your other changes, it seems like our skins are generally now more or less identical except in the format of the css. I mean, there's also that I'm still supporting arc-grey and that you still don't have hover animations on buttons, and your ui is a bit more spaced out than mine. But effectively the skins seem like they're now more or less identical. So at this point I don't see any reason to keep updating mine if you're going to keep updating yours. So would you rather merge the repos, or do nothing? I could put a link to your fork in my readme or something.

**Comment by drakkar1969** [comment]
Sorry, I have been really remiss in replying, my apologies.
I have added back support for the Arc Darker variant because you got me thinking about it being many people's favorite variant - and didn't make much sense to drop support for one of the 3 Arc variants. I had just been lazy about making all the changes required when the addressbar is merged into the titlebar - which I have now done.
I think it still makes sense to keep the two options - mine and yours - available for people to choose from, as there are some differences in styling.
If you're happy to add a link in your fork that would be appreciated :)

**Comment by Tiamarth** [comment]
Sure I can do that, no worries. I'll probably be officially dropping support for the original Arc gtk theme in my variation in the near future as there's no reason for me to support it with your fork in existence. Primarily, you appear to be a much more competent coder and designer than me, so it would be better to direct people to the much more well developed fork. I'll still keep up support for Arc Grey as long as someone better doesn't come along to do that too, though, but my attention has been recently directed to another project anyway.

Also, I appreciate that you've kept my signature in the css around, but if you could remove my email and discord from it at some point, that would be great. I don't want people randomly adding me on Discord or emailing me for support about this skin when it's not really my project anymore. You can feel free to keep or remove as much of the rest of the comment as you want. Obviously I'll never complain about a link to my website existing there ;) but I can't justify claiming ownership over this css when almost none of it was written by me.

And also feel free to close this issue whenever you want, I don't think I have anything more to add to the discussion but will reply if you do. Thanks!

**Comment by drakkar1969** [comment]
Thanks, but not sure I'm that good at the css coding - or the design :).
Good point regarding your contacts, deleted from the css file now.

**Comment by Tiamarth** [comment]
Thanks, very much appreciated

And, you're at least better at it than me as far as I can tell.

I'm going to go ahead and close this issue, feel free to reopen it if you have more to add
